### PR TITLE
fix: Change the reference to old LSP7 interface id to a random value

### PR DESCRIPTION
**Pull request by b00ste**
### What kind of change does this PR introduce (bug fix, feature, docs update, ...)?

fix

### Other information:

Change the reference of old LSP7 interface id  to a random value as it is not relevant for the tests.
- `0xe33f65c3` to `0xdeadbeef`

### CODE REVIEW

**Review by Hugoo — COMMENTED** [review]
These cahnges are on very low level function tests: `TupleKeyValue`

These functions do not care about the content of the tuple. They simply encode/decode. This is not related to any standard.

It happened that we use the test case with values from our standards. But this is really not related. Therefore, I would suggest to simply use totally random value. So no one in the future thinks these tests are related to any LSP / standard, interface ids.

File: src/lib/utils.test.ts [hunk 1]
```diff
@@ -282,8 +282,8 @@ describe('utils', () => {
       {
         valueContent: '(Bytes4,Number)',
         valueType: '(bytes4,bytes8)',
-        encodedValue: '0xe33f65c30000000000000010',
-        decodedValue: ['0xe33f65c3', 16],
+        encodedValue: '0x5fcaac270000000000000010',
+        decodedValue: ['0x5fcaac27', 16],
```
Comment (Hugoo) [inline src/lib/utils.test.ts:286]:
```suggestion
        encodedValue: '0xdeadbeaf0000000000000010',
        decodedValue: ['0xdeadbeaf', 16],
```

File: src/lib/decodeData.test.ts [hunk 2]
```diff
@@ -183,8 +183,8 @@ describe('tuple', () => {
       {
         valueContent: '(Bytes4,Number)',
         valueType: '(bytes4,bytes8)',
-        encodedValue: '0xe33f65c3000000000000000c',
-        decodedValue: ['0xe33f65c3', '12'],
+        encodedValue: '0x5fcaac27000000000000000c',
+        decodedValue: ['0x5fcaac27', '12'],
```
Comment (Hugoo) [inline src/lib/decodeData.test.ts:187]:
```suggestion
        encodedValue: '0xdeadbeaf000000000000000c',
        decodedValue: ['0xdeadbeaf', '12'],
```

**Review by Hugoo — COMMENTED** [review]
lets make it totally different - instead of putting new values from standards.

**Comment by b00ste** [comment]
> lets make it totally different - instead of putting new values from standards.

Tests are passing locally.

**Review by Hugoo — APPROVED** [review]

**Pull request merged**